### ISSUE TITLE
showing edges blanks viewer window

### ISSUE THREAD

**Issue by mngrm3a** [bernhard-42/vscode-ocp-cad-viewer#82]
Vertices and faces work without any issues but edges will blank the viewer.
```python
from build123d import *
from ocp_vscode import *

part = Box(10, 10, 10)

show_object(part)

# show_object(part.vertices() >> Axis.Z)  # works
# show_object(part.edges() >> Axis.Z)  # doesn't work
# show_object(part.faces() >> Axis.Z)  # works
```
I tested this with 2.2.2 and all 3 lines render as expected.

**Comment by bernhard-42** [comment]
@mngrm3a I can only reproduce when `measure_tools`are `True`
Could it be that the issue only occurs when `measure_tools=True`? Similar as in https://github.com/bernhard-42/vscode-ocp-cad-viewer/issues/81

**Comment by mngrm3a** [comment]
Indeed, it works when the measure tools are disabled.

**Comment by bernhard-42** [comment]
Thanks.
Duplicate of https://github.com/bernhard-42/vscode-ocp-cad-viewer/issues/81

**Comment by bernhard-42** [comment]
fixed in 2.4.0
